app/src/main/java/org/listenbrainz/android/service/ListensService.kt

```diff
@@ -15,22 +16,23 @@ import retrofit2.http.Path
 import retrofit2.http.Query
 
 interface ListensService {
-    @GET("1/user/{user_name}/listens")
+    @GET("user/{user_name}/listens")
     suspend fun getUserListens(@Path("user_name") user_name: String, @Query("count") count: Int): Listens
 
     @GET("http://coverartarchive.org/release/{MBID}")
     suspend fun getCoverArt(@Path("MBID") MBID: String): CoverArt
 
-    @GET("1/validate-token")
-    suspend fun checkIfTokenIsValid(@Header("Authorization") token: String?): TokenValidation
+    @POST("submit-listens")
+    @GET("validate-token")
+    suspend fun checkIfTokenIsValid(@Header(AUTHORIZATION) token: String?): TokenValidation
 
-    @POST("1/submit-listens")
-    fun submitListen(@Header("Authorization") token: String?,
+    @POST("submit-listens")
+    fun submitListen(@Header(AUTHORIZATION) token: String?,
                      @Body body: ListenSubmitBody?): Call<ResponseBody?>?
 
-    @GET("1/user/{user_name}/services")
-    suspend fun getServicesLinkedToAccount(
-        @Header("Authorization") authHeader: String?,
+    @GET("user/{user_name}/services")
```

Comment:
What's up with this change? I think we had fixed it.

Reply:
You mean removal of "1/" in URL?

Reply:
Why is this not a suspend function anymore?